alerts/resource_alerts.libsonnet

```
@@ -34,18 +34,34 @@
           } +
           if $._config.showMultiCluster then {
             expr: |||
-              sum(namespace_cpu:kube_pod_container_resource_requests:sum{%(ignoringOverprovisionedWorkloadSelector)s}) by (%(clusterLabel)s) - (sum(kube_node_status_allocatable{%(kubeStateMetricsSelector)s,resource="cpu"}) by (%(clusterLabel)s) - max(kube_node_status_allocatable{%(kubeStateMetricsSelector)s,resource="cpu"}) by (%(clusterLabel)s)) > 0
+              (count(kube_node_info) == 1
```

If `showMultiCluster` is `true` that implies the `cluster` label is available, so the check here should probably use the `cluster` label (so that each cluster is checked on whether it has a single node).

Additionally, I suggest a de-dupe for multiple KSM using `max` like so:

```suggestion
              (count by (cluster) (max by (cluster, node) (kube_node_info)) == 1
```